### PR TITLE
Sync jna version with es/upstream

### PR DESCRIPTION
**Pull request by mfussenegger**

### CODE REVIEW

File: gradle/version.properties [hunk 1]
```diff
@@ -33,7 +33,7 @@ jackson=2.8.10
 log4j=1.2.7
 log4j2=2.9.1
 slf4j=1.6.2
-jna=4.2.2
+jna=4.4.0-1
```
[the remark this answers is not in the document]
Comment (matriv) [reply gradle/version.properties]: It was appveyor that failed in the past: https://github.com/crate/crate/pull/6033

[the remark this answers is not in the document]
Comment (mfussenegger) [reply gradle/version.properties]: It also failed here - which is why I didn't assign anybody to review yet =)

[the remark this answers is not in the document]
Comment (mikethebeer) [reply gradle/version.properties]: @matriv thx for remind. yes, the downgrade was necessary due to a version incompatibility on appveyor.

[the remark this answers is not in the document]
Comment (matriv) [reply gradle/version.properties]: @mfussenegger Maybe I jumped in too fast, sorry :-)
It just came to my mind that we had an issue in the past.

**Pull request closed without merging**